portfolio/src/main/webapp/loginCheck.js

```diff
@@ -0,0 +1,25 @@
+document.addEventListener("DOMContentLoaded", displayPage, false);
+
+function displayPage(e){
+  fetch('/loginStatus').then(response => response.json()).then(data =>{
+    if(data.loginStatus == false){
+      const body = document.getElementById("content");
+      body.innerHTML = "";
+
+      const error = document.createElement("h1");
+      error.appendChild(document.createTextNode("Error: Not logged in!"));
+      body.appendChild(error);
+
+      const loginLink = document.createElement("a");
+      loginLink.href = data.url;
+      loginLink.appendChild(document.createTextNode("Click here to log in."));
+      body.appendChild(loginLink);
+    }
+    else{
+      const logoutLink = document.getElementById("logout");
+      const logoutLinkText =document.getElementById("logoutText");
+      logoutLink.href = data.url;
+      logoutLinkText.innerHTML = "Click here to log out, " + data.email;
```

Comment:
Considering an email address isn't a person's name, but rather an account name, it might make more sense to have this read something like:

Click here to sign out from data.email

or something similar.